routes/land-preview.js

```diff
@@ -0,0 +1,210 @@
+const path = require('path');
+const stream = require('stream');
+const fs = require('fs');
+const url = require('url');
+const querystring = require('querystring');
+const http = require('http');
+const https = require('https');
+const crypto = require('crypto');
+const zlib = require('zlib');
+const child_process = require('child_process');
+const mime = require('mime');
+
+const {getObject, putObject} = require('../aws.js');
+const puppeteer = require('puppeteer');
+const browserManager = require('../browser-manager.js');
+
+const PREVIEW_HOST = '127.0.0.1';
+const PREVIEW_PORT = 8999;

```

Comment:
If this has its own unique server, it should be on a different port from `land.js` or else this won't start.